.clang-format

```diff
@@ -77,6 +77,7 @@ IndentCaseLabels: false
 IndentWidth: 8
 # SpaceBeforeParens: ControlStatementsExceptControlMacros # clang-format >= 13.0
 UseTab: Always
+SortIncludes: false
```

Comment:
Shouldn't the header dependencies be fixed instead? It doesn't seem right that headers need to be included in certain order to work.

Reply:
Agreed - will work on it.